converters/google/vendor_utils.go

```diff
@@ -41,6 +41,14 @@ func getProject(d converter.TerraformResourceData, config *converter.Config, cai
 		} else {
 			log.Printf("[WARN] Failed to retrieve project_id for %s from resource", cai.Name)
 		}
+	case "storage.googleapis.com/Bucket":
```

Comment:
Does a bucket's IAM policy have a project ID, and is that project ID actually important?

Normally we need the project for an IAM policy because it is used to construct the URL/self link for the IAM policy, but in the case of storage I don't think that's true

Reply:
Storage buckets have a project ID. Storage bucket IAM doesn't, but we can get it from the storage bucket if they've been merged.

The only reason this needs to be here is that terraform-validator currently requires it to exist & errors if it's not present. See #206 for more details.